flowmachine/flowmachine/core/query.py

```
@@ -29,7 +29,11 @@
     get_redis,
     submit_to_executor,
 )
-from flowmachine.core.errors.flowmachine_errors import QueryResetFailedException
+from flowmachine.core.errors.flowmachine_errors import (
+    QueryResetFailedException,
+    QueryErroredException,
```

Ruff

`flowmachine.core.errors.flowmachine_errors.QueryErroredException` imported but unused

Remove unused import: `flowmachine.core.errors.flowmachine_errors.QueryErroredException`

(F401)